src/Propel/Generator/Model/Column.php

```diff
@@ -1739,6 +1739,7 @@ public static function generatePhpSingularName(string $phpName): string
     public function isContent(string $content): bool
     {
         $contentAttribute = $this->getAttribute('content');
+        // @todo: expression below has same operands - fix or kick?
 
         return $contentAttribute && strtoupper($contentAttribute) === strtoupper($contentAttribute);
```

Comment:
Oh, it has to be:

```suggestion
        return $contentAttribute && strtoupper($contentAttribute) === strtoupper($content);
```

Good find!